### PR TITLE
Add Generic_KPI options guidance

### PR DESCRIPTION
**Pull request by Reinis-FRP**
Signed-off-by: Reinis Martinsons <reinis@umaproject.org>

### CODE REVIEW

**Review by ChandlerDeKock — APPROVED** [review]
Approved from my side - since we both worked on it.

File: docs/uma-tokenholders/general-kpi.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+---
+title: Launching Generalized KPI Options
```
Comment (smb2796) [inline docs/uma-tokenholders/general-kpi.md:2]: Small nit: the KPI options themselves are not generalized, just the price identifier being used. Maybe change this to "Using the Generalized KPI Options Price Identifier"

File: docs/uma-tokenholders/general-kpi.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+## Ancillary Data Parameters
+
+Below are the parameters that are required when using the generalized KPI options. A short description of the parameters and a few examples illustrate the flexibility in using the generalized KPI option identifier. Please consult [UMIP-117](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-117.md) for ancillary data formatting syntax.
+
+### Metric
+
+A short description and units of the KPI or metric you wish to incentivize. The metric is what is considered the actual number when setting your KPI options. This description should be short and to the point.
+
+#### Metric Examples
```
Comment (smb2796) [inline docs/uma-tokenholders/general-kpi.md:32]: These seem too broad for actual usage, so might not be very good examples. Should probably specify an example protocol/liquidity pool etc.

**Review by smb2796 — APPROVED** [review]
Looks good. Just left two small comments

File: docs/uma-tokenholders/general-kpi.md (hunk 1 above)
Comment (Reinis-FRP) [reply docs/uma-tokenholders/general-kpi.md]: updated

File: docs/uma-tokenholders/general-kpi.md (hunk 2 above)
Comment (Reinis-FRP) [reply docs/uma-tokenholders/general-kpi.md:32]: Updated to UMA specific examples

**Pull request merged**